cwls/chksum_seqval_wf_interleaved_fq.cwl

```diff
@@ -66,15 +71,23 @@ steps:
       post_headers:
         source: post_headers
     out: [chksum_json, post_server_response]
-    run: cgp-chksum.cwl
+    run: https://raw.githubusercontent.com/cancerit/dockstore-cgp-chksum/0.2.0/Dockstore.cwl
```

Comment:
Should this be linked to a version or not?

Reply:
Yes, but I thought "/0.2.0/" part was referring a version.  How can I link it to a version if that's not the correct path?

Reply:
As discussed, the two comments about version number can be ignored. This way is the lesser of two evils.